README.md

```diff
@@ -5,11 +5,11 @@
 Framework Sass &amp; Angular by Lucca
 [luccaSA.github.io/lucca-ui](http://luccaSA.github.io/lucca-ui)
 
-# Version 2.0
+# Version 3.1
```

Comment:
Je viens de tilter mais vu qu'on passer en Version 3.2, il faudrait garder, version 3.1 en mode `bower install lucca-ui` et 3.2 en mode `npm install lucca-ui`

Reply:
Il faut peut être garder la version 2.0 ?

Reply:
if you completely drop bower support, it's breaking change enough to go to version 4.0

Reply:
so if you do it, use this occasaion to update all dependencies (angularjs, ng-ui-bootstrap, ...)